### PR TITLE
1657407 Add descriptions for SEO.

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
## Summary

Add description metadata per [1657407](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1657407)

@rpetrusha and @mairaw, you are down as authors for these in my spreadsheet. Please review when you get the chance. Let me know if somebody else should be looking at these.

### CODE REVIEW

File: docs/framework/install/run-net-framework-1-1-apps.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Run .NET Framework 1.1 apps on Windows 8, Windows 8.1, or Windows 10"
+description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported on many versions of the Windows operating system."
```
Comment (Thraka) [inline docs/framework/install/run-net-framework-1-1-apps.md]: Is this better? Is this correct?

```suggestion
description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported by Microsoft."
```

Comment (Thraka) [inline docs/framework/install/run-net-framework-1-1-apps.md]: Is this better? Is this correct?

```suggestion
description: "Describes how to accommodate apps that require the .NET Framework 1.1 which is no longer supported by Microsoft."
```

File: docs/framework/install/troubleshoot-blocked-installations-and-uninstallations.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Troubleshoot blocked .NET Framework installations and uninstallations"
+description: "Troubleshoot issues you encounter that prevent or block installation of the .NET Framework. Consult blocking messages for information to resolve issues."
```
Comment (Thraka) [inline docs/framework/install/troubleshoot-blocked-installations-and-uninstallations.md]: isn't prevent and block communicating the same thing? I know the article uses it, but the term "blocking message" seems odd and I don't think it's common. Perhaps status is better.

```suggestion
description: "Troubleshoot issues you encounter that prevent installation of the .NET Framework. Consult status messages for information to resolve issues."
```

File: docs/framework/install/guide-for-developers.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Install the .NET Framework developer pack or redistributable"
+description: "Developers can install the .NET Framework or install it with their applications. You can install Developer Pack or targeting pack versions and redistributables."
```
Comment (Thraka) [inline docs/framework/install/guide-for-developers.md]: Maybe something like this?

```suggestion
description: "Developers can download and install the .NET Framework, a developer pack, and a targeting pack. Developers can also include the .NET Framework redistributable with their apps."
```

**Review by Thraka — CHANGES_REQUESTED** [review]
Looks good. Some minor changes to wording suggested. Let me know what you think.

File: docs/framework/install/run-net-framework-1-1-apps.md (hunk 1 above)
Comment (TimShererWithAquent) [reply docs/framework/install/run-net-framework-1-1-apps.md]: I don't see any place in docs that says MS still supports 1.1 in any context. This is probably correct.

File: docs/framework/install/guide-for-developers.md (hunk 3 above)
Comment (TimShererWithAquent) [reply docs/framework/install/guide-for-developers.md]: This is about 10 char over the 160 limit for descriptions. 

Maybe: 

Developers can download and install the .NET Framework developer pack and targeting pack. You can include the .NET Framework redistributable with your apps.

File: docs/framework/install/run-net-framework-1-1-apps.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/install/run-net-framework-1-1-apps.md]: @richlander would probably know.

Comment (TimShererWithAquent) [reply docs/framework/install/run-net-framework-1-1-apps.md]: If @richlander could weigh in -

If we're not sure that some version of windows still runs 1.1, I think we should stay with my original wording.

File: docs/framework/install/guide-for-developers.md (hunk 3 above)
Comment (TimShererWithAquent) [reply docs/framework/install/guide-for-developers.md]: @thraka - let me know if my new wording is OK.

**Comment by TimShererWithAquent** [comment]
@Thraka - Hi Andy - This PR is almost a week old and I'd like to resolve it. 

In guide-for-developers.md, I edited your change down to 160 char. If that's OK, let's go with it.

That leaves wording around support for .NET Framework 1.1. Maybe we can side-step the question of support as follows:

"Describes how to accommodate apps that require the .NET Framework 1.1. In most cases, software vendors should upgrade to a newer version of the .NET Framework."

File: docs/framework/install/guide-for-developers.md (hunk 3 above)
Comment (Thraka) [reply docs/framework/install/guide-for-developers.md]: Sounds good.

**Review by Thraka — APPROVED** [review]
Approved. Thank you for the work on this. I've sent a message out to find out if we can use the "not supported by Microsoft" clause just to clarify.

**Comment by Thraka** [comment]
Confirmed, it's good as it is. Thanks!

**Pull request merged**